config/locale/de.json

```diff
@@ -1,39 +1,204 @@
+	"Oops, you need to specify more than just a city name to locate accurately your position":"Fehler: Der angegebene Standort konnte nicht eindeutig bestimmt oder gefunden werden.",
+	"gen":"gen",
+	"all items":"alle items",
+	"all pokemon":"alle pokemon",
+	"stardust":"sternenstaub",
+	"Stardust":"Sternenstaub",
+	"404 No valid quests found":"Fehler: Kein gültiges Quest angegeben.",
+	"form":"form",
+	"404 no valid tracks found":"Fehler: Kein gültiger Raid-Boss oder gültiges Raid-Level angegeben.",
+	"everything":"alle",
+	"individually":"individuell",
+	"maxlevel":"maxlevel",
+	"great":"super",
+	"greatcp":"superwp",
+	"ultra":"hyper",
+	"ultracp":"hyperwp",
+	"maxcp":"maxwp",
+	"maxiv":"maxiv",
+	"maxweight":"maxgewicht",
+	"maxatk":"maxatk",
+	"maxdef":"maxdef",
+	"maxsta":"maxsta",
+	"cp":"wp",
+	"iv":"iv",
+	"atk":"atk",
+	"def":"def",
+	"sta":"sta",
+	"weight":"gewicht",
+	"404 No monsters found":"Fehler: Kein gültiges Pokémon angegeben.",
+	"Your location is currently set to":"Folgender Standort ist gesetzt:",
+	"You have not set a location yet":"Es wurde noch kein Standort gesetzt.",
+	"Your alerts are currently":"Benachrichtigungen sind aktuell",
+	"enabled":"aktiviert.",
+	"disabled":"deaktiviert.",
+	"You are currently set to receive alarms in":"Folgende Gebiete sind gesetzt:",
+	"You have not selected any area yet":"Es wurde kein Gebiet gesetzt.",
+	"You're tracking the following monsters:":"Folgende Pokémon werden beobachtet:",
+	"You're not tracking any monsters":"Es werden keine Pokémon beobachtet.",
+	"Everything":"_alle_",
+	"any":"_alle_",
+	"greatpvp":"Superliga",
+	"ultrapvp":"Hyperliga",
+	"gender":"Geschlecht",
+	"grunt type":"Rüpel-Typ",
+	"You're tracking the following raids:":"Folgende Raids werden beobachtet:",
+	"You're not tracking any raids":"Es werden keine Raids beobachtet.",
+	"You're tracking the following quests:":"Folgende Quests werden beobachtet:",
+	"You're not tracking any quests":"Es werden keine Quests beobachtet.",
+	"You're tracking the following invasions:":"Folgende Team-GO-Rocket-Invasionen werden beobachtet:",
+	"You're not tracking any invasions":"Es werden keine Team-GO-Rocket-Invasionen beobachtet.",
+	"Tracking list is quite long. Have a look at":"Beobachtungsliste ist zu umfangreich für direkte Ausgabe. Siehe stattdessen:",
+	"Tracking list is long, but Hastebin is also down. ☹️ \nTracking list made into a file:":"Beobachtungsliste ist zu umfangreich für direkte Ausgabe. Sie wird stattdessen als Datei zur Verfügung gestellt:",
+	"Possible weather change at":"Möglicher Wetterwechsel um",
+	"unknown":"unbekannt",
+	"language":"sprache",
+	"area":"gebiet",
+	"egg":"ei",
+	"help":"hilfe",
+	"invasion":"invasion",
+	"location":"standort",
+	"poracle":"poracle",
+	"quest":"quest",
+	"raid":"raid",
+	"start":"start",
+	"stop":"stop",
+	"track":"beobachte",
+	"tracked":"status",
+	"unregister":"abmelden",
+	"untrack":"unbeobachte",
+	"weather":"wetter",
+	"list":"auflisten",
+	"instinct":"Intuition",
```

Comment:
i think this is a parameter for gyms (as with valor and mystic)

Reply:
you're right,... the three team names aren't used in any other place, AFAICS,... so will me lower case in my revised commit